Tasks/DownloadBuildArtifactsV0/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -43,5 +43,7 @@
   "loc.messages.LatestBuildFound": "Latest build found:  %s",
   "loc.messages.LatestBuildNotFound": "Latest build not found",
   "loc.messages.LatestBuildFromBranchNotFound": "Latest build from branch %s not found",
-  "loc.messages.DownloadingContainerResource": "Downloading items from container resource %s"
+  "loc.messages.DownloadingContainerResource": "Downloading items from container resource %s",
+  "loc.messages.DefinitionIDFound": "Definition Name %s resolved to id %d",
+  "loc.messages.InvalidBuildDefinitionID": "Definition ID %s didn't correspond to a valid definition"
```

Comment:
Definition **name** %s correspond to a valid definition?

Reply:
Yes

Reply:
changed